.github/workflows/crowdin-daily.yml

```diff
@@ -0,0 +1,32 @@
+# Download translations from Crowdin and push to GitHub
+
+name: Crowdin Daily Workflow
+
+on:
+  schedule:
+    - cron: '0 0 * * *'  # every day at midnight
+
+jobs:
+  sync:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Setup Java JDK
+        uses: actions/setup-java@v1
+        with:
+          java-version: 1.8
+      - name: Setup Python
+        uses: actions/setup-python@v2
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Crowdin Sync
+        shell: bash
+        env:
+          CROWDIN_API_KEY: ${{ secrets.CROWDIN_API_KEY }}
+        run: |
+          cd intl
+          python3 crowdin_sync.py "$CROWDIN_API_KEY"
+          git config user.name github-actions
+          git config user.email github-actions@github.com
+          git add .
+          git commit -m "Fetch translations from Crowdin"
+          git push
```

Comment:
I am not 100% sure this will work. If I recall correctly, simply pushing a commit like this won't go through.
That's why I used `github-push-action`:

```
      - name: Commit files
        run: |
          git config --local user.email "41898282+github-actions[bot]@users.noreply.github.com"
          git config --local user.name "github-actions[bot]"
          git add intl/*
          git commit -m "Recreate translation source text files" -a
      - name: GitHub Push
        uses: ad-m/github-push-action@v0.6.0
        with:
          github_token: ${{ secrets.GITHUB_TOKEN }}
          branch: ${{ github.ref }}
```

Reply:
From my own testing, this should work. I made a demo repo which add a zero to a text file every ten minutes.
https://github.com/guoyunhe/test-github-actions-commit

Reply:
Ah, well, if you tested this and it worked, then that's great.